private/Interface/python/particle.cxx

```diff
@@ -269,7 +269,16 @@ void init_particle(py::module& m) {
         .def_property_readonly("parent_particle_energy", &Secondaries::GetParentParticleEnergy)
         .def_property_readonly("energy", &Secondaries::GetEnergy)
         .def_property_readonly("time", &Secondaries::GetTime)
-        .def_property_readonly("propagated_distance", &Secondaries::GetPropagatedDistance);
+        .def_property_readonly("propagated_distance", &Secondaries::GetPropagatedDistance)
+        .def_property("entry_point",
+                    &Secondaries::GetEntryPoint,
+                    &Secondaries::SetEntryPoint)
```

Comment:
Why should there be an interface to set entry and exit point by user?

Reply:
Thats correct; just took it from the c lib.
This can both be deleted.